packages/web/components/ui/code-block-node.tsx

```
@@ -96,48 +98,48 @@ function CodeBlockCombobox() {
     <Popover open={open} onOpenChange={setOpen}>
       <PopoverTrigger asChild>
         <Button
-          size="sm"
-          variant="ghost"
-          className="h-6 justify-between gap-1 px-2 text-xs text-muted-foreground select-none"
+          size='sm'
+          variant='ghost'
+          className='h-6 justify-between gap-1 px-2 text-xs text-muted-foreground select-none'
           aria-expanded={open}
-          role="combobox"
+          role='combobox'
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)